Kudu.Core/Tracing/XmlTracer.cs

```diff
@@ -98,7 +101,7 @@ private IDisposable WriteStartElement(string title, IDictionary<string, string>
 
                 strb.Append(new String(' ', _infos.Count * 2));
                 strb.AppendFormat("<step title=\"{0}\" ", XmlUtility.EscapeXmlText(title));
-                strb.AppendFormat("date=\"{0}\" ", DateTime.UtcNow.ToString("yyy-MM-ddTHH:mm:ss.fff"));
+                strb.AppendFormat("date=\"{0}\" ", info.StartTime.ToString("yyy-MM-ddTHH:mm:ss.fff"));
```

Comment:
I know this is the way it was before the PR, but shouldn't it be 4 Ys as in `"yyyy-MM-ddTHH:mm:ss.fff"` vs `"yyy-MM-ddTHH:mm:ss.fff"` like in [here](https://github.com/projectkudu/kudu/pull/1714/files?diff=unified#diff-27786ae8673c1540926cc9debfd30b31L277)